packages/material/src/mui-controls/MuiSelect.tsx

```diff
@@ -50,7 +50,7 @@ export const MuiSelect = React.memo((props: EnumCellProps & WithClassname) => {
       id={id}
       disabled={!enabled}
       autoFocus={appliedUiSchemaOptions.focus}
-      value={data || ''}
+      value={data !== undefined && data !== null ? data : ''}
```

Comment:
```suggestion
      value={data ?? ''}
```

Reply:
I took the same solution as implemented in **MuiInputInteger.tsx**. I'm assuming then that we're targetting TS >= 3.7 with nullish coalescing?

Reply:
Yes we can use the `??` operator here. The other fix is over 2 years old, so that's why it wasn't used there.

However conceptually `null` is a valid value for enums, so what we really should do is just check for `undefined`, i.e.
```suggestion
      value={data !== undefined ? data : ''}
```

Reply:
Thanks. I'll make the change. Out of curiosity, how is `null` rendered?

Reply:
If no label is given via the [`oneOf` `const` construct](https://github.com/eclipsesource/jsonforms/blob/ba6bdc5a93044a872285cd71ed40d9a54f437561/packages/examples/src/enum.ts#L38-L45) we just [stringify the value](https://github.com/eclipsesource/jsonforms/blob/ba6bdc5a93044a872285cd71ed40d9a54f437561/packages/core/src/util/renderer.ts#L162-L165), so it will be shown as `"null"`.